web_monitoring/cli/cli.py

```diff
@@ -827,6 +827,30 @@ def _is_valid(url):
         return False
 
 
+def validate_db_credentials():
+    """
+    Validate Web Monitoring DB credentials by creating a temporary client and
+    attempting to get the current user session.
+
+    Raises
+    ------
+    InvalidCredentials
```

Comment:
This doesn’t match what you’re actually raising. ;)

This description should either change to `Exception` or it *and* the code should change to `UnauthorizedCredentials`.

Reply:
Doh 🤦, will fix in the next commit.